src/lang/proc_macros/controller.rs

```diff
@@ -273,3 +276,30 @@ impl ProcMacroChannels {
         Self { response_sender, response_receiver, error_sender, error_receiver }
     }
 }
+
+enum InitializationFailedInfo {
+    NoMoreRetries { retries: u32, in_minutes: u64 },
```

Comment:
```suggestion
    NoMoreRetries,
```
We don't need these anymore, use consts directly.